type.go

```diff
@@ -304,15 +305,21 @@ func (t *intType) ConvertedType() *deprecated.ConvertedType {
 	return &convertedTypes[convertedType]
 }
 
+// FixedLenByteArray decimals are sized based on precision
+// this function calculates the necessary byte array size
+func CalcDecimalFixedLenByteArraySize(precision int) int {
```

Comment:
Would this function ever need to be called by users of this library? If not, I'd recommend we make it unexported, it's easier to export later if needed than take away and break programs.